__tests__/puppeteer_test.re

```diff
@@ -22,11 +36,14 @@ let noSandbox =
     ()
   );
 
-describe("Puppeteer", () =>
+describe("Puppeteer", () => {
   test("executablePath", () =>
     Puppeteer.executablePath() |> expect |> toContainString("chromium")
-  )
-);
+  );
+  test("defaultArgs()", () =>
+    Puppeteer.defaultArgs() |> expect |> toHaveLength(17)
```

Comment:
This breaks if the number of default arguments change. Maybe we should just check that the length is greater than 0.

Reply:
Thanks! Fixed in commit 4bf5e9f